.github/workflows/qiita-plugin-ci.yml

```diff
@@ -154,5 +154,5 @@ jobs:
     - name: lint
       run: |
         pip install -q flake8
-        flake8 qp_klp setup.py scripts/*
+        flake8 qp_klp
```

Comment:
why `rm scripts/*`?

Reply:
Pyproject.toml is now responsible for creating configure_klp and start_klp binaries. The syntax for creating them requires the contents of the scripts directory to now be part of the module (moved from being co-level with qp_klp directory to being a subdirectory of qp_klp). Since the scripts directory is now under qp_klp, it doesn't need to be tested separately.

Reply:
Got it, thank you for the explanation. Do you foresee any issues with `configure_klp` vs `configure_klp.py` ? 

BTW I'm still confused why this was necessary for this repo and if this is going to be needed for other repos ...

Reply:
There shouldn't be issues with configure_klp vs configure_klp.py, because the installer will generate a separate python script in 'bin' named 'configure_klp' that will call the same config() function in configure_klp.py. 

There isn't a way in pyproject.toml to install scripts the way setup.py does (scripts=['scripts/configure_klp', 'scripts/start_klp']). Instead, there's a section header [project.scripts] in pyproject.toml where you name entry points in your module and little wrapper scripts get created for them in bin.

Reply:
To be more explicit, we could take the entry point functions of configure_klp.py and start_klp.py move the them into the module, and remove these two files entirely and configure_klp and start_klp will still be created in bin.